cloud_utils/slack_utils.py

```diff
@@ -0,0 +1,38 @@
+import traceback
+from typing import Dict, Optional, Text
+
+import gamla
+import requests
+
+
+def send_message_to_webhook(webhook_url: Text, text: Text, data: Optional[Dict]):
+    message = {"text": text}
+    if data is not None:
+        message.update(data)
+    requests.post(webhook_url, json=message)
```

Comment:
We have a gamla function called post_json_async (unless you want to keep this sync)